KestrelMock/Startup.cs

```diff
@@ -24,7 +26,17 @@ public void ConfigureServices(IServiceCollection services)
             services.AddTransient<IResponseMatcherService, ResponseMatcherService>();
             services.AddTransient<IInputMappingParser, InputMappingParser>();
             services.AddTransient<IUriPathReplaceService, UriPathReplaceService>();
-            services.Configure<MockConfiguration>(configuration.GetSection("MockSettings"));
+            services.Configure<MockConfiguration>(opts => 
+            {
+                var mockSettings = configuration
```

Comment:
Initialize the ConcurrencyDictionary by reading the configuration from the provided appsettings file, then inserting each MockSetting into the dictionary.